pom.xml

```diff
@@ -24,7 +24,7 @@
         <rxjava-reactive-streams.version>1.2.1</rxjava-reactive-streams.version>
         <spring.tuple.version>1.0.0.RELEASE</spring.tuple.version>
         <spring.integration.tuple.version>1.0.0.RELEASE</spring.integration.tuple.version>
-        <reactor.version>3.0.5.RELEASE</reactor.version>
```

Comment:
Maybe we can start managing this through their release train BOM (Aluminum, Bismuth, etc).